subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3448,11 +3456,14 @@ static int controller_cmd_handle(uint16_t  ocf, struct net_buf *cmd,
 		le_read_buffer_size(cmd, evt);
 		break;
 
-#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_SYNC_ISO) || \
+	defined(CONFIG_BT_CTLR_CONN_ISO)
 	case BT_OCF(BT_HCI_OP_LE_READ_BUFFER_SIZE_V2):
 		le_read_buffer_size_v2(cmd, evt);
 		break;
-#endif /* CONFIG_BT_CTLR_ADV_ISO || CONFIG_BT_CTLR_CONN_ISO */
+#endif /* CONFIG_BT_CTLR_ADV_ISO || CONFIG_BT_CTLR_SYNC_ISO ||
+	* CONFIG_BT_CTLR_CONN_ISO
+	*/
```

Comment:
Note to myself: Supported commands bitmap needs update accordingly if this PR goes ahead for approval, i.e. host still continues to use the HCI command under Synchronized Receiver only build.